arc

```diff
@@ -0,0 +1,178 @@
+#!/bin/bash
+
+# set -e
```

Comment:
You're not checking the result of all commands, so errors should be catched to avoid having the driver continue when something went wrong.

Reply:
Woops, fixed now!